Description: Fix BytesIO import
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/1022390
Forwarded: no
Last-Update: 2022-11-01

--- python-enmerkar-0.7.1.orig/tests/test_extract.py
+++ python-enmerkar-0.7.1/tests/test_extract.py
@@ -4,7 +4,10 @@ import unittest
 import pytest
 
 from babel.messages import extract
-from babel._compat import BytesIO
+try:
+    from babel._compat import BytesIO
+except ImportError:
+    from io import BytesIO
 
 import django
 from enmerkar.extract import extract_django
